Kickstarter-iOS/AppDelegate.swift

```diff
@@ -496,7 +496,7 @@ extension AppDelegate: UNUserNotificationCenterDelegate {
     withCompletionHandler completionHandler: @escaping (UNNotificationPresentationOptions) -> Void
   ) {
     self.rootTabBarController?.didReceiveBadgeValue(notification.request.content.badge as? Int)
-    completionHandler(.alert)
+    completionHandler([.banner, .list])
```

Comment:
`.alert` was deprecated in iOS 14. The docs say to now use `.banner` and `.list` https://developer.apple.com/documentation/usernotifications/unnotificationpresentationoptions/1649506-alert

`.list` shows notifications in Notification Center.
`.banner` presents notifications as a banner.